src/sentry/static/sentry/app/components/timeSince.jsx

```diff
@@ -38,6 +38,11 @@ class TimeSince extends React.PureComponent {
 
   constructor(props) {
     super(props);
+
+    this.state = {
+      date: '',
+      suffix: props.suffix,
+    };
```

Comment:
You could get rid of the constructor if you just make the state a prop of the class

Reply:
I have removed `suffix` from state. Date has to stay in the state because we update the relative date with a ticker (it counts "1 minute ago", then "2 minute ago", etc.).

Reply:
He means you could change `state` to be a class property and remove `constructor`

eg

```javascript
class Component extends React.Component {
  state = {
    relative: ''
  };
}

Reply:
Thanks @billyvg! Is this the new code style going forward? It's inconsistently applied across the codebase.

Reply:
Good question, it's something we should bring up again in the next frontend TSC